CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 ## CHANGE LOG
 
+### Version 0.0.2 (February 21, 2022)
+* Supports Signed Call Android SDK v0.0.2 and Signed Call iOS SDK v0.0.2.
+* Adds Push Primer support for the [Android 13 notification runtime permission](https://developer.android.com/develop/ui/views/notifications/notification-permission).
```

Comment:
@shivamsharma2710 Mention that SC iOS v0.0.2 fixes the call screen UI issue

Reply:
@darshanclevertap Done. Please review it again.